db-tests/src/test/java/gov/nasa/jpl/aerie/database/DatabaseTestHelper.java

```diff
@@ -20,53 +27,67 @@ public class DatabaseTestHelper {
   private final String dbName;
   private final String appName;
   private final File initSqlScriptFile;
+  private final Optional<Path> migrationsDirectory;
```

Comment:
TODO: Make this better.

Currently - the DatabaseTestHelper has branching logic based on whether migrationsDirectory was provided - if it is provided, it will ignore the `initSqlScriptFile` argument. This is unintuitive and can be better.